src/java.desktop/share/classes/javax/swing/ImageIcon.java

```diff
@@ -178,6 +183,7 @@ public ImageIcon (String filename) {
      * of the image.
      * @param location the URL for the image
      * @param description a brief textual description of the image
+     * @throws {@code NullPointerException} if (@code null) URL is passed.
```

Comment:
```suggestion
     * @throws NullPointerException if a {@code null} URL is passed
```
The exception type is automatically rendered in monospaced font with a link to its description.

You meant to use braces instead of parentheses.

Usually, there's no full stop for `@throws`.

Alternatively, specify the parameter name for more clarity:
```suggestion
     * @throws NullPointerException if {@code location} is {@code null}
```